retis-events/src/raw_packet.rs

```diff
@@ -540,6 +550,115 @@ impl RawPacket {
 
         write!(f, " win {}", tcp.get_window())?;
 
+        if (tcp.get_data_offset() * 4).saturating_sub(20) > 0 {
+            write!(f, " [")?;
+            let mut sep = DelimWriter::new(',');
+
+            for opt in tcp.get_options_iter() {
+                let optnum = opt.get_number().0;
+                let datalen = match opt.get_length_raw().first() {
+                    Some(len) if *len >= 2 => *len as usize - 2,
+                    _ => 0,
+                };
+
+                sep.write(f)?;
+                write!(
+                    f,
+                    "{}",
+                    match optnum {
+                        0 => "eol",
+                        1 => "nop",
+                        2 => "mss",
+                        3 => "wscale",
+                        4 => "sackOK",
+                        5 => "sack",
+                        6 => "echo",
```

Comment:
Are we noting things to add to `pnet` somewhere?
If so, list += add extra options to https://docs.rs/pnet_packet/0.35.0/pnet_packet/tcp/TcpOptionNumbers/index.html

Reply:
Can we extend our internal pnet to declare these numbers?

Reply:
Yep, `retis-pnet` is there for those things.